core/src/ast_builder/expr/between.rs

```diff
@@ -0,0 +1,36 @@
+use super::ExprNode;
+
+impl ExprNode {
+    pub fn between<T: Into<Self>>(self, negated: bool, low: T, high: T) -> Self {
```

Comment:
How about splitting `between` into two - `between` and `not_between`?
Then we don't need to take `negated` as a param.
Then...
```rust
let actual = col("num").between(1, 10);
let expected = "num BETWEEN 1 AND 10";
test_expr(actual, expected);
```

```rust
let actual = col("num").not_between(1, 10);
let expected = "num NOT BETWEEN 1 AND 10";
test_expr(actual, expected);
```

Reply:
There was a parameter called negated , so I thought I should get this as a value as well. As you said, I will modify the function and make a Pull Request again later. Thanks a lot

Reply:
According to the existing requirements, it has been divided into two functions, the 'between' function and the 'not_between' function.